FlowCal/excel_ui.py

```diff
@@ -216,7 +217,8 @@ def process_beads_table(beads_table,
     This function processes the entries in `beads_table`. For each row, the
     function does the following:
         - Load the FCS file specified in the field "File Path".
-        - Transform the forward scatter/side scatter channels if needed.
+        - Transform the forward scatter/side scatter and fluorescence
+          channels to a.u.
```

Comment:
maybe transform to *RFI* (same vein as other comments: more precise and enforcing serialization on RFI)